getting_started/step_by_step/signals.rst

```diff
@@ -326,6 +419,18 @@ reaches 0.
 
     var health = 10
 
+ .. code-tab:: csharp C#
+
+    using Godot;
+
+    public class CustomSignal : Node2D
+    {
+        [Signal]
+        public delegate void HealthDepleted();
+
+        private int Health = 10;
```

Comment:
The C# naming convention for fields is camelCase.

```suggestion
        private int health = 10;
```

In the engine we also prefix private fields with an underscore `_` but I'm not sure if that also applies to the documentation.

Reply:
Made it with a lower case letter, without an underscore sign. 

In my opinion, it's better to have it with `_`, but it won't be consistent with other documentation.

Reply:
Reverted back to the uppercase variant to match it with the current docs. As well as other variables, like Speed and AngularSpeed.